Find a functionally equivalent code:android.graphics.Paint_Delegate.measureText:COMMENT
Method Modifier: 
Comment:/*package*/

@@ -1,3 +1,3 @@
 {
-    return new BidiRenderer(null, this, text).renderText(index, index + count, isRtl, null, 0, false, 0, 0);
+    return new BidiRenderer(null, this, text).renderText(index, index + count, bidiFlags, advances, advancesIndex, false);
 }

